lib/core/DeviceManagerPlugin.ts

```diff
@@ -35,10 +35,11 @@ import { DeviceManagerEngine } from "./DeviceManagerEngine";
 import { DecodersRegister } from "./registers/DecodersRegister";
 import { InternalCollection } from "./InternalCollection";
 import { ModelsRegister } from "./registers/ModelsRegister";
+import { KuzzleRole } from "lib/modules/shared/types/KuzzleRole";
 
 export class DeviceManagerPlugin extends Plugin {
   public config: DeviceManagerConfiguration;
-  public roles = {};
+  public roles: KuzzleRole[] = [];
```

Comment:
As you update the type of this class property (from object to array), should we need information about already deployed environments migration process? Maybe this is not useful, I don't realize the implications of this update

Reply:
This property is only used inside the plugin so there will be no impact